processing/src/test/java/org/apache/druid/math/expr/FunctionTest.java

```diff
@@ -391,6 +391,73 @@ public void testArrayContains()
     assertExpr("array_contains([1, null, 2], null)", 1L);
     assertExpr("array_contains([1, null, 2], [null])", 1L);
     assertExpr("array_contains([1, 2], null)", 0L);
+    assertExpr("array_contains([1, 2, 3], [])", 1L);
+    assertExpr("array_contains([], [])", 1L);
+    assertExpr("array_contains([null], [])", 1L);
+    assertExpr("array_contains(['foo', 'bar'], [])", 1L);
+    assertExpr("array_contains(null, [])", null);
+  }
+
+  @Test
+  public void testMvContains()
```

Comment:
can you add some tests for what happens if other types of columns make it in here, like scalar numbers and number arrays and stuff? i think should be fine, just might be nice to have.. see there are tests for scalar strings which is :+1: just would like to see a bit more (and also maybe as bindings since different paths for singleThreaded for constant and dynamic args)

Reply:
Added some tests. I didn't use bindings, but I don't think they're needed, because `assertExpr` tests both with and without single-threaded mode. Using constants should let both modes be exercised.